index.html

```diff
@@ -14,13 +14,24 @@
     <header class="header backgrnd-clr">
       <nav class="navbar">
         <a href="index.html"><img src="images/icons/Welcome.png" alt="logo" /></a>
-        <img src="images/icons/Union.png" alt="hamburger-menu" />
+        <button class="bars" type="button"><img class="bars-image" src="images/icons/Union.png" alt="hamburger-menu" /></button>
         <ul class="navlist">
```

Comment:
when in desktop version your hamburger menu should not be visible 
you can solve this by giving it aa style of `visibility:hidden` in your CSS

your desktop design 
![image](https://user-images.githubusercontent.com/60612329/197069322-c0c98221-328d-4e80-9f26-b1a809001672.png)